Makefile

```diff
@@ -52,7 +57,7 @@ sub-flow: $(venv)
 ## deploy basic_flow to kubernetes
 kubes-deploy: export PREFECT_API_URL=http://localhost:4200/api
 kubes-deploy: $(venv)
-	docker compose build app && docker compose push app
+	docker-compose build app && docker-compose push app
```

Comment:
on macOS the versions of these differ:
```
$ docker-compose --version
docker-compose version 1.29.2, build 5becea4c

$ docker compose version
Docker Compose version v2.6.0
```

The latest is the new golang version of docker compose which is faster and has more features.

Are you on linux? On linux you can integrate docker-compose into the docker cli [like this](https://github.com/tekumara/setup-ubuntu/blob/main/install-root/docker.sh#L18), so `docker compose` works... perhaps this is worth mentioning in the docs, rather than changing all the commands to docker-compose, which downgrades the experience for macOS users.